__tests__/app.test.js

```diff
@@ -45,3 +45,30 @@ describe("GET /api/categories", () => {
 			});
 	});
 });
+
+describe("GET /api/reviews", () => {
+	test("status:200, responds with a reviews array", () => {
+		return request(app)
+			.get("/api/reviews")
+			.expect(200)
+			.then(({ body }) => {
+				const { reviews } = body;
+				expect(reviews).toBeInstanceOf(Array);

```

Comment:
You can delete line 56 as this is not necessary. As you are using a forEach, we know this is an array already.